src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const shortTerm = 2;
+  const midTerm = 6;
+  const dailyFee = 40;
+  const midTermDiscount = 20;
+  const longTermDiscount = 50;
```

Comment:
for this kind o variables please use uppercase names - [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) point 2